devenv-cli.sh

```diff
@@ -1,4 +1,4 @@
-SOLVCON_WORKING_ROOT=${HOME}/solvcon/devenv
+SOLVCON_DEVENV_SRC=${HOME}/solvcon/devenv
```

Comment:
The name is a little bit long for typing.  How about using `SCDE` (SOLVCON dev env) and making it `SCDE_SRC`?

Reply:
+1. Updated the PR.

Reply:
`SOLVCON_DEVENV_SRC` has been renamed as `SCDE`.